tableauserverclient/server/endpoint/groups_endpoint.py

```diff
@@ -57,28 +71,12 @@ def create(self, group_item):
     @api(version="2.0")
     def remove_user(self, group_item, user_id):
         self._remove_user(group_item, user_id)
```

Comment:
At this point it may not even be necessary to have the `_remove_user` function since we don't do anything special now